tensorrt_llm/_torch/visual_gen/models/ltx2/pipeline_ltx2_two_stages.py

```
@@ -39,13 +39,45 @@
 # LoRA helpers
 # ---------------------------------------------------------------------------
 
 
+def _get_free_gpu_memory_gib() -> Optional[float]:
+    """Return free memory on the current CUDA device, or ``None`` if unavailable."""
+    if not torch.cuda.is_available():
+        return None
+
+    try:
+        free_bytes, _ = torch.cuda.mem_get_info()
+    except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)